cli/common.h

```diff
@@ -62,10 +62,22 @@ enum switchtec_fw_type check_and_print_fw_image(int img_fd,
 			" * an index (0, 1, 2)\n" \
 			" * a PCI address (3:00.1)\n" \
 			" * an I2C path with slave address (/dev/i2c-1@0x20)\n" \
-			" * a UART path (/dev/ttyUSB0)\n" \
```

Comment:
No UART in MFG? What happens if the user specifies a UART?

Reply:
The command will fail with error code if UART is specified.

Reply:
What causes that error code? And is the error message actually informative? I don't remember seeing any code to that effect...

Reply:
Since BL1/2 does not support GAS register access, the command will fail during switchtec_open(), with error message: 'Error communicating with the device. Please check your setup.'

In MAIN, the command runs with no problem. However, we don't recommend user using UART for mfg commands.

Reply:
Well that's hardly informative. It sounds like they just have a bad connection.

It should error out with the message: "Error: manufacturing commands are not supported over UART connections". (or something like that).

Reply:
Good idea. Updated. 

The way to determine if a path is _tty_ might be clumsy. Suggestions welcome!